src/app/volume/container/volume-details.container.ts

```diff
@@ -57,4 +70,10 @@ export class VolumeDetailsContainerComponent extends WithUnsubscribe() implement
       });
   }
 
+  public setDiskOfferingColumns() {
+    const configParams = this.configService.get('diskOfferingParameters');
+    if (configParams) {
+      this.diskOfferingColumns = this.diskOfferingColumns.concat(configParams);
+    }
+  }
```

Comment:
I suppose we can get columns in diskOfferingEffects, do not we? Or is this way more suitable for us?

Reply:
I noticed you added a new action to disk offerings: LOAD_DEFAULT_DISK_PARAMS_REQUEST. Why do not you use it here?